### DISCUSSION TITLE
在xdusetup修改字体参数后报错

### DISCUSSION THREAD

**Discussion by neilwen987** [note286/xduts#77] [Q&A] [answered]
如题，在修改字体参数之前编译正常，按照xduts.pdf教程修改字体设置后，再次尝试编译报错. 这是啥原因捏？
![image](https://user-images.githubusercontent.com/56498265/226159768-8b51a7de-545f-48c5-8bed-a6d796b23453.png)

**Comment by sikouhjw** [comment, marked as answer]
> 如题，在修改字体参数之前编译正常，按照xduts.pdf教程修改字体设置后，再次尝试编译报错. 这是啥原因捏？

改了什么？怎么改的？

**Reply by note286** [reply]
这么一看，我文档里

> xdufont 宏包可以搭配任意文档类进行使用

确实不严谨，xdufont 就不能搭配 xduts 中其他文档类使用。有空修修，先记下来。

**Reply by note286** [reply]
你竟然成为了答案！

**Reply by sikouhjw** [reply]
> 你竟然成为了答案！

6

**Reply by note286** [reply]
拿本本记好标准答案，下次用。

> 改了什么？怎么改的？

**Comment by note286** [comment]
首先，请你认真阅读文档，看一下`xdufont`是干啥的。

其次，Overleaf基本没什么内置字体，如果你想使用其他字体，请自行上传，字体名称已经在文档中给出了，怎么调用文档中也给出了。

**Reply by sikouhjw** [reply]
> 其次，Overleaf基本没什么内置字体，如果你想使用其他字体，请自行上传，字体名称已经在文档中给出了，怎么调用文档中也给出了。

感觉可以加个 Overleaf 内思源宋体的选项？

**Reply by note286** [reply]
主要吧，思源字体不全，没有宋楷黑仿全套，缺的我就不知道搭配啥好，很纠结，就一直没弄。

**Reply by neilwen987** [reply]
![image](https://user-images.githubusercontent.com/56498265/226160534-0f361c89-6cd0-42fa-b321-752d8adb4963.png)
按照文档内说明调用的，然后报错了

**Reply by note286** [reply]
我已经回复你了，Overleaf没有这些字体，需要你上传，我已经很贴心的把所需要的字体文件名列出来了。

**Reply by note286** [reply]
不知道你是否注意到了`style/font-type`，以及下面的话。

**Reply by sikouhjw** [reply]
> 主要吧，思源字体不全，没有宋楷黑仿全套，缺的我就不知道搭配啥好，很纠结，就一直没弄。

当然是全部用 Overleaf 自带的（比没得用好）：https://sikouhjw.github.io/2021/02/14/2021-02-14-fontset-overleaf/

**Reply by note286** [reply]
我记一下，最近忙完搞一下，脑子里有不少想法，折腾折腾。
